database/postgres/event_delivery.go

```diff
@@ -390,9 +390,9 @@ func (e *eventDeliveryRepo) FindEventDeliveriesByIDs(ctx context.Context, projec
 		return nil, err
 	}
 
-	query = e.db.Rebind(query)
+	query = e.db.GetDB().Rebind(query)
 
-	rows, err := e.db.QueryxContext(ctx, query, args...)
+	rows, err := e.db.GetDB().QueryxContext(ctx, query, args...)
```

Comment:
```suggestion
	rows, err := e.db.GetReadDB().QueryxContext(ctx, query, args...)
```

Reply:
This method is used by `ForceResendEventDeliveriesService`. Does it matter if there is a replication lag here?

Reply:
No, there isn't; we can afford some lag here.

Reply:
Ok then.